src/alias_registry.hh

```diff
@@ -22,7 +22,7 @@ public:
     {
         auto merge = [](auto&& first, const AliasMap& second) {
             return concatenated(std::forward<decltype(first)>(first)
-                                | filter([&second](auto& i) { return not second.contains(i.key); }),
+                                | filter([&second](const auto& i) { return not second.contains(i.key); }),
```

Comment:
Making iterators constant in utility functions is nice but it's a lot of work, it clutters the code slightly, and also not really a big deal. Can't really undo it now that you've fixed them all, but it's a practice that will probably not be observed in future code :p

Reply:
I'm ok with that. :)